app/views/people/show.erb

```diff
@@ -1,6 +1,7 @@
 <!-- Main jumbotron for a primary marketing message or call to action -->
 <div class="jumbotron">
     <h1>Hello <%= current_user.name %>!</h1>
+  <p></p><%= current_user.measurements.latest.weight %> kg </p>
```

Comment:
I am thinking: I have to rethink the current_measurement and the `latest` scope, because basically, I am afraid they are exactly the same right now. And, I will add this point to the thinking fest.